src/networks.ts

```diff
@@ -0,0 +1,21 @@
+import { ETH_RPC, ETH_RPC_MAINNET } from '@big-whale-labs/constants'
+import Network from 'models/Network'
+import createProvider from 'helpers/providers/createProvider'
+import env from 'helpers/env'
+
+export default {
+  [Network.Mainnet]: {
+    network: Network.Mainnet,
+    ...createProvider(Network.Mainnet, {
+      default: ETH_RPC_MAINNET,
```

Comment:
I think we shouldn't use `@bwl/constants` here, use `env.ETH_RPC_MAINNET`, it will have `@bwl/contants` + your own `.env` if you need it

Reply:
Most likely we need to make a separate variable for these purposes